src/store/RootState.ts

```diff
@@ -1,4 +1,13 @@
-import QueryRepresentation from '@/sparql/QueryRepresentation';
+export default interface RootState {
+	conditionRow: ConditionRow;
+}
 
-export interface RootState extends QueryRepresentation {
+export interface ConditionRow {
+	propertyData: {
```

Comment:
I was about to question the `Data` suffix on both `propertyData` and `valueData` but it makes sense as soon as the aforementioned meta data is also in here :+1: